Library/Formula/openal-soft.rb

```diff
@@ -40,10 +31,7 @@ class OpenalSoft < Formula
   def install
     ENV.universal_binary if build.universal?
 
-    # Please don't reenable example building. See:
```

Comment:
I'd be surprised if this wasn't still broken. Please do:

```
brew install sdl2 sdl_sound
```
And then install this build locally. Let us know if that passes.